packages/core/src/actions/send.ts

```diff
@@ -123,10 +123,12 @@ function execute(
   const { to, event } = params;
 
   actorContext.defer(() => {
-    to.send(
+    actorContext?.system.sendTo(
```

Comment:
By using `system.sendTo`, we make it possible to track the source/target of events.